video_creation/screenshot_downloader.py

```diff
@@ -73,7 +73,7 @@ def get_screenshots_of_reddit_posts(reddit_object: dict, screenshot_num: int):
         print_substep("Launching Headless Browser...")
 
         browser = p.chromium.launch(
-            headless=True
+            headless=False
```

Comment:
```suggestion
            headless=True
```
Why would we set headless to False? Probably forgot to turn it back off when debugging :)

Reply:
I was not sure if it was working headless, will check and If it works I'll change it back to True

Reply:
Wont work in headless

Reply:
> Wont work in headless

Why not? Are you sure? Reddit shouldn't be able to see that so I think it won't make a difference

Reply:
Reddit knows somehow